pkg/apis/rbac/v1alpha1/types.go

```diff
@@ -28,21 +28,25 @@ import (
 // 3. deny by default
 
 // PolicyRule holds information that describes a policy rule, but does not contain information
-// about who the rule applies to or which namespace the rule applies to.
+// about who the rule applies to or which namespace the rule applies to. Rules can either apply
+// to API resources (such as "pods" or "secrets") or non-resource URL paths (such as "/api"),
+// but not both.
 type PolicyRule struct {
 	// Verbs is a list of Verbs that apply to ALL the ResourceKinds and AttributeRestrictions contained in this rule.  VerbAll represents all kinds.
 	Verbs []string `json:"verbs" protobuf:"bytes,1,rep,name=verbs"`
 	// AttributeRestrictions will vary depending on what the Authorizer/AuthorizationAttributeBuilder pair supports.
 	// If the Authorizer does not recognize how to handle the AttributeRestrictions, the Authorizer should report an error.
 	AttributeRestrictions runtime.RawExtension `json:"attributeRestrictions,omitempty" protobuf:"bytes,2,opt,name=attributeRestrictions"`
```

Comment:
Our TPM component is actually a custom admission controller which uses this field to do it's own authorization. The RBAC API group just happens to be an extremely convenient way of associating policies with users or groups, even when the RBAC authorizer is disabled.